### PR TITLE
feat/CCN3-2025/confirmation-emails-and-page-templates

### PR DESCRIPTION
**Pull request by LRydeard**
## Description

Changes logic for sending confirmation emails and getting templates.

## Todo's before PR

- [ ] Branch has been rebased against develop
- [ ] Unit tests passed (`make test`)
- [ ] Selenium tests have passed
- [ ] PR named appropriately - see [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [ ] PR description describes the overall goals of PR commits
- [ ] Templates have been checked against the relevant user-story

## Migrations

- [ ] Null fields in models specifies default value
- [ ] You generated and applied migrations (`make migrate`)
- [ ] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [ ] Specified reviewers (even if it's yourself)
- [ ] Specified assignees (those who'll merge)
- [ ] Specified labels

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [ ] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

**Comment by LRydeard** [comment]
Apologies for the branch name, this is unintended but correct.

File: application/tests/test_declaration/test_form_validation.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def test_unselected_declaration_boxes_raise_form_error(self):
+    """
+    Test that not selecting each declaration box raises the respective error.
+    """
+    pass
```
Comment (birchbit) [inline application/tests/test_declaration/test_form_validation.py]: There should probably be a `skipTest` here (or implement it!)

File: application/tests/test_declaration/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from .test_form_validation import *
+from .test_helper_methods import *
+from .test_routing import *
```
Comment (birchbit) [inline application/tests/test_declaration/__init__.py]: What is the reason for importing * from the inner modules? I've encountered cases in one of the other projects where this has been polluting the namespace to the point where names are being shadowed

Comment (LRydeard) [reply application/tests/test_declaration/__init__.py]: For tests, I think there's no need. I copied the dunder file from another test module.
I'll remove these to make sure.

File: application/tests/test_declaration/test_form_validation.py (hunk 1 above)
Comment (LRydeard) [reply application/tests/test_declaration/test_form_validation.py]: I think this was a test I un-commented, I'll skipTest it for now.

**Pull request merged**